dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestLineInterface.cpp

```diff
@@ -183,10 +183,15 @@ TEST(DataInterfaceTest, Line) {
   ASSERT_FALSE(li2.getInitialConnected2());
   ASSERT_EQ(li2.getR(), 0.0);
   ASSERT_EQ(li2.getX(), 0.01);
+  ASSERT_DOUBLE_EQ(li2.getP1(), 0.0);
+  ASSERT_DOUBLE_EQ(li2.getP2(), 0.0);
+  ASSERT_DOUBLE_EQ(li2.getQ1(), 0.0);
+  ASSERT_DOUBLE_EQ(li2.getQ2(), 0.0);
 
   MySecondLine.getTerminal1().setP(std::numeric_limits<double>::infinity());
-  MySecondLine.getTerminal2().setP(-4444) ;
+  MySecondLine.getTerminal2().setP(-4444);
   MySecondLine.getTerminal1().setQ(444.4);
+  MySecondLine.getTerminal2().setQ(std::nan("not  a number"));
```

Comment:
the NaN case is already tested above. I would like a test with a finite number for each P1/Q1/P2/Q2.

Reply:
done now...